pkg/operation/botanist/component/fluentoperator/crds.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fluentoperator
+
+import (
+	"context"
+	_ "embed"
+
+	"github.com/gardener/gardener/pkg/client/kubernetes"
+	"github.com/gardener/gardener/pkg/operation/botanist/component"
+	"github.com/gardener/gardener/pkg/utils/flow"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	//go:embed assets/crd-fluentbit.fluent.io_clusterfilters.yaml
+	clusterFilterCRD string
+	//go:embed assets/crd-fluentbit.fluent.io_clusterfluentbitconfigs.yaml
+	clusterFBConfigCRD string
+	//go:embed assets/crd-fluentbit.fluent.io_clusterinputs.yaml
+	clusterInputCRD string
+	//go:embed assets/crd-fluentbit.fluent.io_clusteroutputs.yaml
+	clusterOutputCRD string
+	//go:embed assets/crd-fluentbit.fluent.io_clusterparsers.yaml
+	clusterParserCRD string
+	//go:embed assets/crd-fluentbit.fluent.io_fluentbits.yaml
+	fluentBitCRD string
+
+	resources []string
+)
+
+func init() {
+	resources = append(resources,
+		clusterFilterCRD,
+		clusterFBConfigCRD,
+		clusterInputCRD,
+		clusterOutputCRD,
+		clusterParserCRD,
+		fluentBitCRD,
+	)
+}
+
+type fluentOperatorCRDs struct {
```

Comment:
The package is already called `fluentoperator`, no need to mention it again

```suggestion
type crds struct {
```